public/app/plugins/panel/graph/align_yaxes.ts

```diff
@@ -137,17 +137,17 @@ function getRate(yLeft: AxisSide, yRight: AxisSide): number {
   }
 
   if (checkOneSide(yLeft, yRight)) {
-    const absLeftMin = Math.abs(yLeft.min);
-    const absLeftMax = Math.abs(yLeft.max);
-    const absRightMin = Math.abs(yRight.min);
-    const absRightMax = Math.abs(yRight.max);
-    const upLeft = _.max([absLeftMin, absLeftMax]);
-    const downLeft = _.min([absLeftMin, absLeftMax]);
-    const upRight = _.max([absRightMin, absRightMax]);
-    const downRight = _.min([absRightMin, absRightMax]);
-
-    const rateLeft = downLeft ? upLeft / downLeft : upLeft;
-    const rateRight = downRight ? upRight / downRight : upRight;
+    const absLeftMin: number = Math.abs(yLeft.min);
+    const absLeftMax: number = Math.abs(yLeft.max);
+    const absRightMin: number = Math.abs(yRight.min);
+    const absRightMax: number = Math.abs(yRight.max);
+    const upLeft: number = _.max([absLeftMin, absLeftMax]) || 0;
+    const downLeft: number = _.min([absLeftMin, absLeftMax]) || 0;
+    const upRight: number = _.max([absRightMin, absRightMax]) || 0;
+    const downRight: number = _.min([absRightMin, absRightMax]) || 0;
```

Comment:
Math.max()
Math.min()

The || 0 part seems weird

Reply:
Yeah, the lodash version might return 'undefined' by it's type definition, so in case it did, default to 0... But I have replaced as per your suggestion.